teuthology/lock/query.py

```diff
@@ -1,6 +1,10 @@
 import logging
 import os
-import urllib
+
+try:
```

Comment:
We could use `six` which is already in requirements.txt to have less code. And later it's easier to grep for it to remove py2 support. Or is six not wanted?

Reply:
I would prefer to avoid `six`, if only because it's yet another thing I would have to learn how to use.